README.md

```diff
@@ -259,33 +260,39 @@ api.upload_maven_component(
   version: VERSION,
   repository: REPOSITORY_NAME,
   tag: TAG,                     # Optional
+  content: FILE_CONTENT,        # Optional
```

Comment:
Can you rename `content` to `remote_filename`, `upstream_filename`, or something along those lines? I think `content` is pretty misleading.

Reply:
And add a description of the field to the comment above this method description.